waspc/data/Generator/templates/react-app/src/api.ts

```diff
@@ -8,35 +8,35 @@ const api = axios.create({
   baseURL: config.apiUrl,
 })
 
-const WASP_APP_AUTH_TOKEN_NAME = 'authToken'
+const WASP_APP_AUTH_SESSION_ID_NAME = 'sessionId'
 
-let authToken = storage.get(WASP_APP_AUTH_TOKEN_NAME) as string | undefined
+let appSessionId = storage.get(WASP_APP_AUTH_SESSION_ID_NAME) as string | undefined
```

Comment:
I've now renamed it to be `waspAppAuthSessionId`, I wanted to avoid using `sessionId` since in the functions below e.g. `setSessionId` the input argument is named `sessionId` and it would shadow this variable.

Before we had `authToken` and `token`, now we have `waspAppAuthSessionId` and `sessionId`

Reply:
Ok I like it!